internal/bulk.go

```diff
@@ -153,6 +157,9 @@ func processRequest(ctx context.Context, req BulkRequest, seq int64) *BulkRespon
 		if len(bld.Template) > 0 {
 			opts.Template = bytes.NewReader(bld.Template)
 		}
+		if opts.PrivateKey == nil {
+			opts.PrivateKey = b.DefaultPrivateKey
+		}
```

Comment:
Not sure if there's a more idiomatic way of assigning the default key when no explicit key is given in the request

Reply:
That looks correct! Obviously not as easy as `opts.PrivateKey ||= b.DefaultPrivateKey` :-)